apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-custom-post-template-actions-click.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+const TRACKABLE_TAG_NAMES = [ 'BUTTON', 'SELECT' ];
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_custom_post_template_save`.
+ *
+ * @returns {{handler: Function, selector: string, type: string}} event object definition.
+ */
+export default () => ( {
+	selector: '.edit-post-sidebar .components-panel__body',
+	type: 'click',
+	handler: ( event ) => {
+		if ( TRACKABLE_TAG_NAMES.includes( event.target.tagName ) ) {
+			tracksRecordEvent( 'wpcom_block_editor_custom_post_template_actions_click' );
+		}
+	},
+} );
```

Comment:
For some reason the **Edit** link is not triggering this event. That's why the description doesn't include the **Edit** link tracking. But I've added a separate event for tracking when editing is activated. Which is basically the same as clicking on the **Edit** link right now.